supernode/server/task_bridge.go

```diff
@@ -34,3 +43,41 @@ func (s *Server) deleteTask(ctx context.Context, rw http.ResponseWriter, req *ht
 	rw.WriteHeader(http.StatusOK)
 	return nil
 }
+
+func (s *Server) registerTask(ctx context.Context, rw http.ResponseWriter, req *http.Request) (err error) {
+	reader := req.Body
+	request := &types.TaskCreateRequest{}
+	if err := json.NewDecoder(reader).Decode(request); err != nil {
+		return errors.Wrap(errortypes.ErrInvalidValue, err.Error())
+	}
+	if err := request.Validate(strfmt.NewFormats()); err != nil {
+		return errors.Wrap(errortypes.ErrInvalidValue, err.Error())
+	}
+
+	resp, err := s.TaskMgr.Register(ctx, request)
+	if err != nil {
+		logrus.Errorf("failed to register task %+v: %v", request, err)
+		return err
+	}
+	logrus.Debugf("success to register task %+v", request)
+	return EncodeResponse(rw, http.StatusOK, &types.ResultInfo{
```

Comment:
use `http.StatusCreated`  instead

In addition, why wrap the result with `types.ResultInfo`. And it's inconsistent with the definition of swagger.

Reply:
> In addition, why wrap the result with `types.ResultInfo`. And it's inconsistent with the definition of swagger.
I do with the response just like the v0.3 bridge.all the  post response do as the format.and what `And it's inconsistent with the definition of swagger.` mean?